action.yml

```
@@ -71,10 +71,10 @@ inputs:
     description: 'A value of tmt.context variable https://tmt.readthedocs.io/en/latest/spec/context.html, variables separated by ;'
     required: false
     default: ''
-  artifacts_url:
-    description: 'Baseurl where results can be found'
+  artifacts_name:
```

```suggestion
  tf_scope:
```
What about renaming to `tf_scope`?

---

@fernflower We are not using this at all. Are you fine with this name?

---

Sure